archetypes/carbon-bundle-archetype/src/test/resources/projects/bundle/reference/src/main/java/org/test/bundle/Greeter.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.test.bundle;
+
+import java.util.logging.Logger;
+
+/**
+ * This class greets.
+ *
+ * @since 1.0.0
```

Comment:
This is not the kernel version. This is the version of the sample project generated by the archetype, therefore it has to be 1.0.0.